pages/_app.js

```diff
@@ -1,15 +1,35 @@
 import GlobalStyle from "../styles";
 import useLocalStorageState from "use-local-storage-state";
 import { useRouter } from "next/router";
+import { useState } from "react";
 
 export default function App({ Component, pageProps }) {
   /* define the state and the initial data from your array by setting defaultValue to travel. use localstoragestate to keep the entries persistent*/
   const [currentTrips, setCurrentTrips] = useLocalStorageState("trips", {
     defaultValue: [],
   });
 
+  /* creating and removing of participants in the form .js */
+  const [participants, setParticipants] = useState([]);
+
+  function submitParticipants(newParticipant) {
```

Comment:
you can only submit one participant with this function, please use the singular `submitParticipant`